File: src/app/models/Rentals.ts
import {Payments} from "./Payments";
export class Rentals{
   
	filmTitle:string;
	rentalDate:string;
	returnDate:string;
	filmId:string;
    rentalId:string;
    staffId:string;
     payments:Payments[];
     
     constructor(filmTitle:string, rentalDate:string, returnDate:string, filmId:string,
        rentalId:string, staffId:string, payments:Payments[]){
            this.filmTitle = filmTitle;
            this.rentalDate = rentalDate;
            this.returnDate = returnDate;
            this.filmId = filmId;
            this.rentalId = rentalId;
            this.staffId = staffId;
            this.payments = payments;
     }

     
}

File: src/main/java/com/microservice/document/Rentals.java
package com.microservice.document;

import java.util.List;

import org.springframework.data.mongodb.core.mapping.Field;

public class Rentals {
	
	@Field("Film Title")
	private String filmTitle;
	@Field("Rental Date")
	private String rentalDate;
	@Field("Return Date")
	private String returnDate;
	private int filmId;
	private int rentalId;
	
	private int staffId;
	@Field("Payments")
	private List<Payments> payments;
	public String getFilmTitle() {
		return filmTitle;
	}
	public void setFilmTitle(String filmTitle) {
		this.filmTitle = filmTitle;
	}
	public String getRentalDate() {
		return rentalDate;
	}
	public void setRentalDate(String rentalDate) {
		this.rentalDate = rentalDate;
	}
	public String getReturnDate() {
		return returnDate;
	}
	public void setReturnDate(String returnDate) {
		this.returnDate = returnDate;
	}
	
	public int getFilmId() {
		return filmId;
	}
	public void setFilmId(int filmId) {
		this.filmId = filmId;
	}
	public int getRentalId() {
		return rentalId;
	}
	public void setRentalId(int rentalId) {
		this.rentalId = rentalId;
	}
	public int getStaffId() {
		return staffId;
	}
	public void setStaffId(int staffId) {
		this.staffId = staffId;
	}
	public List<Payments> getPayments() {
		return payments;
	}
	public void setPayments(List<Payments> payments) {
		this.payments = payments;
	}
	
	
	

}
